platform-hub-api/app/mailers/announcement_mailer.rb

```diff
@@ -0,0 +1,14 @@
+class AnnouncementMailer < ApplicationMailer
+
+  include ActionView::Helpers::TextHelper
+
+  def announcement_email announcement, recipients
+    @announcement = announcement
```

Comment:
Maybe change that to `@announcement_text = announcement.text` for consistency and use it directly in template?